lib/token-assert.js

```diff
@@ -239,42 +260,25 @@ TokenAssert.prototype.differentLine = function(options) {
  * @param {Object} [options.message]
  * @param {Object} [options.atLeast] At least how many lines the tokens are apart
  * @param {Object} [options.atMost] At most how many lines the tokens are apart
- * @param {object} [options.exactly] Exactly how many lines the tokens are apart
+ * @param {Object} [options.exactly] Exactly how many lines the tokens are apart
  */
 TokenAssert.prototype.linesBetween = function(options) {
     var token = options.token;
     var nextToken = options.nextToken;
+    var atLeast = options.atLeast;
+    var atMost = options.atMost;
+    var exactly = options.exactly;
 
     if (!token || !nextToken) {
         return;
     }
 
-    if (token === nextToken) {
-        throw new Error('You cannot specify the same token as both token and nextToken');
-    }
-
-    var atLeast = options.atLeast;
```

Comment:
It looks like these are still used down on 312, did some other logic change that I'm not noticing that makes this work?